docs/api-docs/channels/channels-extending-existing.md

```diff
@@ -1,47 +1,61 @@
 # Extending Sales Channel Apps with Channels Toolkit
 
+<!-- https://developer.bigcommerce.com/api-docs/channels/tutorials/extend-existing-apps -->
+
 <div class="otp" id="no-index">
 
-### On this page
-- [Step 1: update API credentials](#step-1-update-api-credentials)
-- [Step 2: integrate channel API](#step-2-integrate-channel-api)
-- [Step 3: Migrate existing data](#step-3-migrate-existing-data)
-- [App requirements](#app-requirements)
-- [Sample configuration](#sample-configuration)
-- [Terminology](#terminology)
-- [Related resources](#related-resources)
+## On this page
+
+- [Extending Sales Channel Apps with Channels Toolkit](#extending-sales-channel-apps-with-channels-toolkit)
+  - [Step 1: Update API credentials](#step-1-update-api-credentials)
+  - [Step 2: Integrate channel API](#step-2-integrate-channel-api)
+    - [All partners](#all-partners)
+    - [Channel manager example](#channel-manager-example)
+    - [Select partners](#select-partners)
+    - [Channel app import section](#channel-app-import-section)
+    - [Channel app settings section](#channel-app-settings-section)
+  - [Step 3: Migrate existing data](#step-3-migrate-existing-data)
+  - [App requirements](#app-requirements)
+    - [Storefronts](#storefronts)
+    - [Marketplaces and marketing](#marketplaces-and-marketing)
+  - [Sample configuration](#sample-configuration)
+  - [Terminology](#terminology)
+  - [Related resources](#related-resources)
+    - [Articles](#articles)
+    - [Endpoints](#endpoints)
 
 </div>
 
-This article provides a guide to  partners who would like to update (or replace) their existing sales channel apps to leverage new functionality available via Channels Toolkit.
+This article provides a guide to partners who would like to update (or replace) their existing sales channel apps to leverage new functionality available via Channels Toolkit.
 
-## Step 1: update API credentials
+## Step 1: Update API credentials
 
 Existing sales channel apps need app credentials with updated OAuth scopes to authenticate and authorize requests to Channels Toolkit APIs. Login to the [Developer Portal](https://devtools.bigcommerce.com) to update your app's credentials. Channel APIs require the following scopes:
 
-|UI Name|Parameter|Enables|
-|-|-|-|
-|Channels Settings Modify|`store_channel_settings`|Creating channels that reference external platforms|
-|Channels Listings Modify|`store_channel_listings`|Creating and reading product listing information|
+| UI Name                  | Parameter                | Enables                                             |

```

Comment:
It is probably best to leave the markdown table as is. I would remove the extra space.